models/parachute_small/model.config

```
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<model>
+  <name>Parachute small</name>
+  <version>1.0</version>
+  <sdf version="1.5">model.sdf</sdf>
+
```

You can add the license as:
```suggestion
  <license>Apache License, Version 2.0</license>
```